verilog/formatting/formatter_test.cc

```diff
@@ -395,8 +395,8 @@ static constexpr FormatterTestCase kFormatterTestCases[] = {
      " `AINIT(S, (D == 4 && K inside {0, 1}) ||"
      " (D == 3 && K== 4))\n",
      "`AINIT(S,\n"
-     "       (D == 4 && K inside {0, 1}) || (\n"
-     "           D == 3 && K == 4))\n"},
+     "       (D == 4 && K inside {0, 1}) ||\n"
+     "           (D == 3 && K == 4))\n"},
```

Comment:
FYI: this has been added as a test for https://github.com/chipsalliance/verible/issues/351. The new behavior seems to be better.

Reply:
+1